### PR TITLE
Add filter to remove private carrier routes

### PR DESCRIPTION
**Pull request by lxhughes**
Asana: https://app.asana.com/0/1148853526253437/1201220680602259/f

Manually identifies the private carrier routes by route ID and filters them out of GTFS pull of routes.

### CODE REVIEW

File: lib/schedule/gtfs/route.ex [hunk 1]
```diff
@@ -71,6 +71,18 @@ defmodule Schedule.Gtfs.Route do
     row["route_type"] == "3"
   end
 
+  @spec bus_route_mbta?(Csv.row()) :: boolean
+  def bus_route_mbta?(row) do
```
[the remark this answers is not in the document]
Comment (lemald) [reply lib/schedule/gtfs/route.ex:75]: Similarly, `bus_route_row?` is now only called internally, so it should probably also be private.

**Review by lemald — CHANGES_REQUESTED** [review]
Basic logic looks good but I have a few comments.

File: lib/schedule/gtfs/route.ex [hunk 2]
```diff
@@ -71,6 +71,18 @@ defmodule Schedule.Gtfs.Route do
...
+    # Verify that route number is not one of the private carriers: 710, 712, 713, 714, 716
+    row["route_id"] not in ["710", "712", "713", "714", "716"]
+  end
+
+  @spec bus_route_validrow?(Csv.row()) :: boolean
+  def bus_route_validrow?(row) do
```
[the remark this answers is not in the document]
Comment (lxhughes) [reply lib/schedule/gtfs/route.ex]: fixed in a440ae99

File: lib/schedule/gtfs/route.ex (hunk 1 above)
[the remark this answers is not in the document]
Comment (lxhughes) [reply lib/schedule/gtfs/route.ex]: fixed in a440ae99

**Review by lemald — APPROVED** [review]

**Pull request merged**